package.json

```diff
@@ -39,16 +40,20 @@
     "@types/jszip": "^3.4.1",
     "@types/xml2js": "^0.4.14",
     "eslint": "^9.8.0",
+    "eslint-config-prettier": "^9.1.0",
     "eslint-formatter-junit": "^8.40.0",
     "jsdom": "^25.0.1",
+    "prettier": "^3.4.2",
     "typescript": "^5.5.4",
     "typescript-eslint": "^8.0.1",
-    "vitest": "^2.0.5"
+    "vitest": "^2.1.8"
```

Comment:
Ok, but shouldn't that be enforced by the yarn lock file?

Reply:
It was just my preference to see that vitest has latest version. I can roll it back.

Reply:
No, don't. I just wanted to understand why.

Reply:
Just to be sure that we use the latest version, it is kind of paranoia.